tests/components/zha/common.py

```diff
@@ -161,23 +161,22 @@ def make_attribute(attrid, value, status=0):
     return True
 
 
-def make_entity_id(domain, device, cluster, use_suffix=True):
-    """Make the entity id for the entity under testing.
+async def find_entity_id(domain, zha_device, hass):
+    """Find the entity id under the testing.
 
     This is used to get the entity id in order to get the state from the state
     machine so that we can test state changes.
     """
-    ieee = device.ieee
-    ieeetail = "".join([f"{o:02x}" for o in ieee[:4]])
-    entity_id = "{}.{}_{}_{}_{}{}".format(
-        domain,
-        slugify(device.manufacturer),
-        slugify(device.model),
-        ieeetail,
-        cluster.endpoint.endpoint_id,
-        ("", "_{}".format(cluster.cluster_id))[use_suffix],
-    )
-    return entity_id
+    ieeetail = "".join([f"{o:02x}" for o in zha_device.ieee[:4]])
+    head = f"{domain}." + slugify(f"{zha_device.name} {ieeetail}")
+
+    entities = hass.states.async_entity_ids(domain)
```

Comment:
These are `entity_ids` and not `entities`, right? I suggest renaming the variable.

Reply:
Yep, will do. Remnants of `hass.states.all()`